### PR TITLE
Connect with Github

### PR DESCRIPTION
**Pull request by harshmangalam**
https://github.com/commune-os/weird/assets/583842/04316c64-6bb8-4fc4-bff9-d95cf86c7093

### CODE REVIEW

**Comment by harshmangalam** [comment]
I have read the CLC Document and I hereby sign the CLC

File: src/routes/(app)/account/github/Repository.svelte [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+	import type { GithubRepo } from '$lib/types/github';
+
+	export let repo: GithubRepo;
+</script>
+
+<article class={`rounded-md border  border-gray-300 p-4 dark:border-gray-700`}>
+	<div class={`justify-beeen flex items-center`}>
```
Comment (zicklag) [inline src/routes/(app)/account/github/Repository.svelte:8]: Should this be `justify-between` or is it not needed?

**Review by zicklag — COMMENTED** [review]
Awesome work, thanks!

CI just noticed one typo.

**Comment by harshmangalam** [comment]
> Awesome work, thanks!
> 
> CI just noticed one typo.

Yes that was tailwind syntax issues , got resolved

**Review by zicklag — APPROVED** [review]

**Pull request merged**